src/rev_ai/apiclient.py

```diff
@@ -337,95 +337,144 @@ def get_list_of_jobs(self, limit=None, starting_after=None):
 
         return [Job.from_json(job) for job in response.json()]
 
-    def get_transcript_text(self, id_):
+    def get_transcript_text(self, id_, group_channels_by=None, group_channels_threshold_ms=None):
```

Comment:
i would use type hints if possible

Reply:
I don't see we use type hints in this code. I consider this to be python 2.x compatibility